### DISCUSSION TITLE
Is it possible to change alphaMode in a model after loading?

### DISCUSSION THREAD

**Discussion by archeo56** [google/model-viewer#2153] [Q&A]
Hello,

Using examples I learn how change color and Material Metalness and Roughness Factors.
I need masking a part of a 3d model and a simple solution could be to change "alphaMode " of the material to "MASK".  I try to modify examples but without successs : I can't set alphamode to  "MASK". Is it possible?
Thank you

**Comment by elalish** [comment]
This isn't available in our scene-graph API yet, but you can do it in our editor and export the resulting glTF: https://modelviewer.dev/editor/

**Reply by archeo56** [reply]
Thank you very much for your answer. I want to hide the object on the fly by user action. I'll try another way. Model-viewer is for me a very good solution that I use extensively on my website.
